### DISCUSSION TITLE
splink requirements on databricks

### DISCUSSION THREAD

**Discussion by hunsicke** [moj-analytical-services/splink#2146] [Q&A]
When I installed splink on a Databricks cluster previously I found a page that listed the spark/python versions and package versions that were needed for the previous version to run properly.  I can't figure out where that page is to update for the new version of Splink.  Can someone please point me in the right direction?
Thanks!

**Comment by RobinL** [comment]
I'm not sure there is one, because we don't have access to databricks ourselves.  there are various discussions that may help e.g. this one
https://github.com/moj-analytical-services/splink/discussions/2114
https://github.com/moj-analytical-services/splink/discussions/1461

**Reply by hunsicke** [reply]
Thanks, Robin, it isn't so much a databricks thing here--just if there is a page where the requirements for installation are listed?

**Reply by RobinL** [reply]
This is the lockfile used for testing the latest version:
https://github.com/moj-analytical-services/splink/blob/master/poetry.lock

You can do something like `poetry export -f requirements.txt --output requirements.txt` to get a requirements.txt

Or else install splink in a fresh venv and do a `pip freeze > requirements.txt`

**Reply by hunsicke** [reply]
Brilliant, thanks!
